test/v6/lossClaims/delete/def1/Def1_DeleteLossClaimValidatorSpec.scala

```diff
@@ -29,49 +29,75 @@ class Def1_DeleteLossClaimValidatorSpec extends UnitSpec {
-        val result: Either[ErrorWrapper, DeleteLossClaimRequestData] = validator(validNino, validClaimId).validateAndWrapResult()
+      "passed a valid request" in {
+        val result: Either[ErrorWrapper, DeleteLossClaimRequestData] = validator(validNino, validClaimId, validTaxYearClaimedFor).validateAndWrapResult()
         result shouldBe Right(
-          Def1_DeleteLossClaimRequestData(parsedNino, parsedClaimId)
+          Def1_DeleteLossClaimRequestData(parsedNino, parsedClaimId, parsedTaxYearClaimedFor)
         )
       }
     }
 
     "return NinoFormatError error" when {
-      "given an invalid nino" in {
-        val result = validator(invalidNino, validClaimId).validateAndWrapResult()
+      "passed an invalid nino" in {
+        val result = validator(invalidNino, validClaimId, validTaxYearClaimedFor).validateAndWrapResult()
         result shouldBe Left(
           ErrorWrapper(correlationId, NinoFormatError)
         )
       }
     }
 
-    "return LossIdFormatError error" when {
-      "given an invalid claim ID" in {
-        val result = validator(validNino, invalidClaimId).validateAndWrapResult()
+    "return ClaimIdFormatError error" when {
+      "passed an invalid claim ID" in {
+        val result = validator(validNino, invalidClaimId, validTaxYearClaimedFor).validateAndWrapResult()
         result shouldBe Left(
           ErrorWrapper(correlationId, ClaimIdFormatError)
         )
       }
     }
 
+    "return TaxYearClaimedForFormatError error" when {
+      "passed an incorrectly formatted taxYearClaimedFor" in {
+        val result = validator(validNino, validClaimId, "202324").validateAndWrapResult()
+
+        result shouldBe Left(ErrorWrapper(correlationId, TaxYearClaimedForFormatError))
+      }
+    }
+
+    "return a RuleTaxYearNotSupportedError error" when {
+      "passed a taxYearClaimedFor before the minimum supported" in {
+        validator(validNino, validClaimId, "2017-18").validateAndWrapResult() shouldBe
+          Left(ErrorWrapper(correlationId, RuleTaxYearNotSupportedError))
+      }
+    }
+
+    "return a RuleTaxYearRangeInvalidError error" when {
+      "passed a taxYear spanning an invalid tax year range" in {
+        val result = validator(validNino, validClaimId, "2020-22").validateAndWrapResult()
+
+        result shouldBe Left(ErrorWrapper(correlationId, RuleTaxYearRangeInvalidError))
+      }
+    }
+
     "return multiple errors" when {
-      "given a request with multiple errors" in {
-        val result = validator(invalidNino, invalidClaimId).validateAndWrapResult()
+      "passed a request with multiple errors" in {
+        val result = validator(invalidNino, invalidClaimId, "invalidTaxYear").validateAndWrapResult()
         result shouldBe Left(
-          ErrorWrapper(correlationId, BadRequestError, Some(List(ClaimIdFormatError, NinoFormatError)))
+          ErrorWrapper(correlationId, BadRequestError, Some(List(ClaimIdFormatError, NinoFormatError, TaxYearFormatError)))
```

Comment:
This should fail so please update to the specific format error `TaxYearClaimedForFormatError`, thanks